### PR TITLE
feat: dag get cli

### PR DESCRIPTION
**Pull request by kumavis**
Here is the skeleton for the `dag` cli command, and an implementation of `dag get <ref>` where `ref` is { `<cid>` or `<cid>/<path>` }.

Not sure what the formatting for output should be, just logging the result as-in for now.

Was unable to run the test due to some unrelated bug on test start, but should be correct.

### CODE REVIEW

File: test/cli/test-dag.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+'use strict'
+
+const expect = require('chai').expect
+const repoPath = require('./index').repoPath
+const describeOnlineAndOffline = require('../utils/on-and-off')
+const ipfs = require('../utils/ipfs-exec')(repoPath)
+
+describe('dag', () => {
+  describeOnlineAndOffline(repoPath, () => {
+    it('get', () => {
```
Comment (VictorBjelkholm) [inline test/cli/test-dag.js]: Should probably have a test-case as well when we try to resolve a hash that does not exists.

Comment (kumavis) [inline test/cli/test-dag.js]: Yeah I wanted to write more extensive tests but I couldn't even get them to run bc of some issue with a networking module

**Comment by diasdavid** [comment]
@kumavis I was rebasing master on this branch to then realise this PR was made to a old branch and not against master.

Mind rebasing  and PR'ing against master?

**Comment by kumavis** [comment]
replaced by https://github.com/ipfs/js-ipfs/pull/753

**Pull request closed without merging**